src/client.rs

```diff
@@ -438,9 +468,6 @@ async fn on_room_message(
             if event.sender.localpart() == user_id.localpart() {
                 return;
             }
-            room.read_receipt(&event.event_id)
```

Comment:
Why removing this?

Reply:
it was panicking, will revisit this later in another PR.

Reply:
Let's add a TODO here if that's the case

Reply:
I think it's more than just that line so I filed this issue #70.